pkg/config/v2/route.go

```diff
@@ -280,6 +281,13 @@ type RouterMatch struct {
 	Headers []HeaderMatcher `json:"headers,omitempty"` // Match request's Headers
 }
 
+// RedirectAction represents the redirect response parameters
+type RedirectAction struct {
```

Comment:
Need support `scheme_redirect`，forexample
`http://mosn.io`  -> `https://mosn.io` or `https://mosn.io`  -> `http://mosn.io` 

refer: https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#envoy-v3-api-field-config-route-v3-redirectaction-scheme-redirect

Reply:
@wangfakang  Got it, but how could we know the current scheme? Perhaps I have missed something, I am unable to find any related headers or methods to get the current scheme.

Reply:
> @wangfakang Got it, but how could we know the current scheme? Perhaps I have missed something, I am unable to find any related headers or methods to get the current scheme.

Could add a 'scheme' variable to get current request `scheme`, here is a var [refer](https://github.com/mosn/mosn/blob/master/pkg/stream/http/var.go).